app/src/main/java/org/eyeseetea/malariacare/domain/entity/ScoreType.java

```diff
@@ -24,14 +24,15 @@
 
 public class ScoreType {
 
-    public enum Classification {HIGH, MEDIUM, LOW}
+    public enum Classification {HIGH, MEDIUM, LOW, NO_RESPONSE}
```

Comment:
NO_RESPONSE is an incorrect name because score 0 is when the score calculation result is 0 and this is possible with no responses or negative responses. A better name would be NO_SCORE